Neos.Eel/Classes/Utility.php

```diff
@@ -36,12 +36,36 @@ public static function getDefaultContextVariables(array $configuration)
                 }
                 $currentPathBase = & $currentPathBase[$pathName];
             }
-            $currentPathBase = new $objectType();
-        }
 
+            if (strpos($objectType, '::')) {
+                if (strpos($variableName, '.')) {
+                    throw new Exception(sprintf('Function helpers are only allowed on root level, "%s" was given?', $variableName), 1557911015);
+                }
+                $currentPathBase = self::createClosureFromConfiguration($objectType);
+            } else {
+                $currentPathBase = new $objectType();
+            }
+        }
         return $defaultContextVariables;
     }
 
+    /**
+     * Create a closure to be used as Helper for eel.
+     *
+     * @param $objectConfiguration string className followed by two colone and the method name
+     * @return callable
+     */
+    private static function createClosureFromConfiguration(string $objectConfiguration): callable
+    {
+        list($className, $methodName) = explode('::', $objectConfiguration, 2);
```

Comment:
Why do we destructure the return of `explode` here just to glue it together into an array inside the closure?

```suggestion
        $classAndMethodName = explode('::', $objectConfiguration, 2);
        return function (...$arguments) use ($classAndMethodName) {
            return call_user_func_array($classAndMethodName, $arguments);
        }
```

Reply:
I simply find it easier to understand that way.

Reply:
Hm, yeah. It's a bit more explicit indeed. Fine for me, thanks for explaining :)